<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
color: white;</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Doctor Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Designation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>License</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctorname">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="designation">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="license">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_number">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="address">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="doctorsubmitbutton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 248, 248);
color: black;
font-size: 30pt;
font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gobackdoctor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 248, 248);
color: black;
font-size: 30pt;
font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
